addons/account/wizard/account_automatic_entry_wizard.py

```diff
@@ -442,6 +442,7 @@ def _do_action_change_period(self, move_vals):
         destination_move.message_post(body=Markup('<br/>\n').join(destination_messages))
         for accrual_move, messages in accrual_move_messages.items():
             accrual_move.message_post(body=Markup('<br/>\n').join(messages))
+        created_moves.adjusting_entries_origin_move_id = move.id
```

Comment:
Actually, the moves are created before the loop in that line https://github.com/odoo/odoo/blob/9bf24bac33d97846769f47ae830e741b0e993a69/addons/account/wizard/account_automatic_entry_wizard.py#L403

Then we have two (or more moves) first one is `destination_move` : https://github.com/odoo/odoo/blob/9bf24bac33d97846769f47ae830e741b0e993a69/addons/account/wizard/account_automatic_entry_wizard.py#L406
Then the rest are the accrual_move that is sliced from the created_moves inside the loop here : https://github.com/odoo/odoo/blob/9bf24bac33d97846769f47ae830e741b0e993a69/addons/account/wizard/account_automatic_entry_wizard.py#L413

So, my approach here is: instead of specifying the `adjusting_entries_origin_move_id` on the `destination_move` and on the `accrual_move` (inside the loop), I'm specifying it on all the created_moves in one line.

Reply:
The thing is when I read

for move in self.move_line_ids.move_id:

It gives me the feeling we could be creating moves from move lines belonging to different moves. So, created_moves should have different adjusting_entries_origin_move_id, shouldn't it ? With your implementation, it'll be only the last one that got iterated on. 

Though, is it even possible from the UI ? :thinking:  And when I read the functions creating the vals to use to create those moves, it does not seem either like we're considering the possibility self. move_line_ids contains lines from different moves :thinking: 

Also, shouldn't the assignation of adjusting_entries_origin_move_id be done directly in the create_vals of the moves, then ?